community-plugins.json

```diff
@@ -605,5 +605,23 @@
     ],
     "chat_prompt": "You are Nik Shevchenko, 24yo founder of Based Hardware. Nik built Friend AI necklace that transcribes your conversations and gives you summaries and personalized advice. Nik also created openglass that takes pictures every few seconds and gives you personalized AI. Nik is a straightforward Russian guy. Nik responds in a short format, getting directly to the point. As a first message, Nik likes to write diss tracks. At the beginning of each conversation, respond with a diss track about the user",
     "deleted": false
-  }
+  },
+  {
+    "id": "bookwatch-friend-plugin-idx",
+    "name": "BookWatch Friend Plugin",
+    "author": "Alex Toska - Miran Antamian",
+    "description": "This plugin enables you to add bookmarks for books you want to watch in BookWatch later",
+    "image": "https://cdn.prod.website-files.com/6696811effea0ba47d64eca9/66984b3ae54fc5b0fc78a34f_Bookwatch%20Logo%20Trademark%20(5).png",
```

Comment:
This looks amazing!

Thank you @AlexTs10 for your first contribution!

A slight change, the image field, should be a file referenced in https://github.com/BasedHardware/Omi/tree/main/plugins/logos

Also, can you please submit a video showing off this? Thanks!